CONTRIBUTING.md

```diff
@@ -127,11 +127,13 @@ Here are some things to keep in mind as you file pull requests to fix bugs, add
 
 ### Release process
 
-- Make sure the tests pass with `yarn test`
+- Make sure you are on the tip of the `main` branch of the Forge repo (`git checkout main && git fetch && git pull`).
```

Comment:
1. Is `git fetch` necessary? I thought `git pull` implied `git fetch`, especially on the default branch.
2. You should be able to use `git switch main` instead of `git checkout main` on sufficiently new git (which should be most installed git versions, unless you're on an ancient machine).

Reply:
Thanks for the catch. I haven't used the raw git commands for those in a while so I forgot. 😓